src/ws/cockpit.socket.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Cockpit Web Service Socket
+Documentation=man:cockpit-ws(8)
+
+[Socket]
+ListenStream=9090
+ExecStartPost=@bindir@/ln -sf @sysconfdir@/@PACKAGE@/active.issue @sysconfdir@/@PACKAGE@/cockpit.issue
+ExecStopPost=@bindir@/ln -sf @sysconfdir@/@PACKAGE@/inactive.issue @sysconfdir@/@PACKAGE@/cockpit.issue
```

Comment:
Please make both commands non-fatal by prefixing with `-`. This is not essential for the socket operation, and we shouldn't assume that we can always write into /etc (read-only system images).

We can not assume that the source files actually exist as long as we ship them in /etc (/usr/ would be fine). Unfortunately `ln` does not have an option to only create the link if the target file actually exists, so in that case we would get a dangling symlink (and I don't think we should invoke a shell here to test first). So let's ignore that case for now.